### PR TITLE
BZ2072523: Use single domain network for all nodes

### PR DESCRIPTION
**Pull request by xenolinux**
[Bug](https://bugzilla.redhat.com/show_bug.cgi?id=2072523)
Preview: [Networking requirements](https://deploy-preview-44391--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-vsphere-installer-infra-requirements_installing-vsphere-installer-provisioned)
QE contact @jinyunma 
OCP version: 4.6+

### CODE REVIEW

**Comment by jinyunma** [comment]
LGTM

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 1]
```diff
@@ -249,7 +249,7 @@ Available resources vary between clusters. The number of possible clusters withi
 [id="installation-vsphere-installer-infra-requirements-networking_{context}"]
 == Networking requirements
 
-You must use DHCP for the network and ensure that the DHCP server is configured to provide persistent IP addresses to the cluster machines.
+You must use DHCP for the network and ensure that the DHCP server is configured to provide persistent IP addresses to the cluster machines. All nodes must be in the same VLAN. You cannot scale the cluster using second VLAN as a Day 2 operation.
```
Comment (nalhadef) [inline modules/installation-vsphere-installer-infra-requirements.adoc:252]: "...using a second..."

**Review by nalhadef — COMMENTED** [review]
Very minor change needed in Line 252

File: modules/installation-vsphere-installer-infra-requirements.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/installation-vsphere-installer-infra-requirements.adoc]: Addressed

**Pull request merged**